### PR TITLE
testlapack: rework eigenvector checks in DgeevTest and adjust tolerances

### PR DESCRIPTION
**Pull request by vladimir-ch**
Please take a look.

### CODE REVIEW

File: lapack/testlapack/dgeev.go [hunk 1]
```diff
@@ -658,80 +643,79 @@ func testDgeev(t *testing.T, impl Dgeever, tc string, test dgeevTest, jobvl lapa
...
-				}
+	// Check that the columns of VL and VR are eigenvectors that:
+	//  - correspond to the computed eigenvalues
+	//  - have Euclidean norm equal to 1
+	//  - have the largest component real
+	bi := blas64.Implementation()
+	if jobvr == lapack.RightEVCompute {
+		resid := residualRightEV(test.a, vr, wr, wi)
+		if resid > vecTol {
+			t.Errorf("%v: unexpected right eigenvectors; residual=%v, want<=%v", prefix, resid, vecTol)
```
Comment (kortschak) [inline lapack/testlapack/dgeev.go]: Currently prefix uses the `%v` verb for the `jobvr` and `jobvl` values. This should probably be `%c` for ease of reading, but that would be another PR since there are a bunch of them and they should all change together.

Comment (vladimir-ch) [reply lapack/testlapack/dgeev.go]: Ok, I'll put it on my todo list.
